transcom/mymove
# B 18798 county lookup on address creation int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18798)

## Summary

This work is to add counties for addresses on creation. There is a later story to take care of populating the county column on address UPDATES, that is [B-18799](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18799).

To test:
1. Create a service member. After creating the profile, you should have a county for the `residential_address_id` and `backup_mailing_address_id` on the service member.
2. Create a shipment (HHG or PPM) for that service member. You should have a county for `pickup_address_id` and for `destination_address_id`.
3. Submit the move as the service member.
4. Services counseling/TOO - as soon as you update the move as the services counselor or TOO, the county column on the shipment addresses will become null because you are updating the shipment. That county value persisting on update will be done as part of B-18799 mentioned above.
5. Get the move through the services counseling step and the TOO review step. 
6. When the move is available to the prime, go to the shipment you just created in the database.
7. Manually reenter the county for the shipment pickup and the destination addresses.
8. Request origin SIT as the prime.
9. Verify that the origin SIT service items have counties in the addresses for `sit_origin_hhg_original_address_id` and `sit_origin_hhg_actual_address_id`.
10. Go back to the shipment and if the county columns are null for pickup or destination addresses, manually reenter the county. 
11. Request destination SIT as the prime.
12. Verify that the destination SIT service items have counties in the addresses for `sit_destination_final_address_id`.
